webrtc.html

```diff
@@ -8929,7 +8929,12 @@ <h2 id="sec.stats-model">Statistics Model</h2>
               <p>The <dfn><code>timestamp</code></dfn>, of type
               <code>DOMHighResTimeStamp</code> [[!HIGHRES-TIME]], associated
               with this object. The time is relative to the UNIX epoch (Jan 1,
```

Comment:
Related to https://github.com/w3c/webrtc-stats/issues/134 - can you replace "associated with this object" with "that indicates the time at which these values were measured"?